src/components/Modal.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+type Props = { isActive: boolean; onClick: () => void };
+
+class Modal extends React.Component<Props> {
+    render() {
+        const isActive = this.props.isActive;
+        return (
+            <div>
+                <div className={`modal ${isActive ? 'is-active' : null}`}>
```

Comment:
You can just check the prop field here `this.props.isActive`

Also, I would replace null with empty string '', it looks more consistent for the ternary operator to return the same type in the true or false scenario.